### PR TITLE
Fix history issue when closing more info dialog by clicking update

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

This caused duplicated history and user had to click multiple time to back button to go back.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/entities/editor-tabs/settings/entity-settings-helper-tab.ts [hunk 1]
```diff
@@ -16,7 +16,6 @@ import {
 } from "../../../../../data/entity_registry";
 import { HELPERS_CRUD } from "../../../../../data/helpers_crud";
 import { showConfirmationDialog } from "../../../../../dialogs/generic/show-dialog-box";
-import { hideMoreInfoDialog } from "../../../../../dialogs/more-info/show-ha-more-info-dialog";
```
Comment (bramkragten) [inline src/panels/config/entities/editor-tabs/settings/entity-settings-helper-tab.ts:19]: Can we remove this function? I think it is not longer needed?

**Review by bramkragten — DISMISSED** [review]

File: src/panels/config/entities/editor-tabs/settings/entity-settings-helper-tab.ts (hunk 1 above)
Comment (piitaya) [reply src/panels/config/entities/editor-tabs/settings/entity-settings-helper-tab.ts:19]: It's used in ha-config-entities but I'm not sure why :

```ts
  public disconnectedCallback() {
    super.disconnectedCallback();
    hideMoreInfoDialog(this);
  }
```

Comment (bramkragten) [reply src/panels/config/entities/editor-tabs/settings/entity-settings-helper-tab.ts:19]: It used to be 2 different dialogs, but they no longer are

Comment (piitaya) [reply src/panels/config/entities/editor-tabs/settings/entity-settings-helper-tab.ts:19]: Ok I will remove it 🙂

**Review by bramkragten — APPROVED** [review]

**Pull request merged**